pkg/apis/apps/v1alpha1/types.go

```diff
@@ -417,6 +421,22 @@ type ApplicationSyncStatus struct {
 	Name                string                                `json:"name,omitempty"`
 	KustomizationStatus *kustomizev1beta2.KustomizationStatus `json:"kustomizationStatus,omitempty"`
 	HelmReleaseStatus   *helmv2beta1.HelmReleaseStatus        `json:"HelmReleaseStatus,omitempty"`
+	RolloutStatus       RolloutStatus                         `json:"rolloutStatus,omitempty"`
```

Comment:
This is different from fluxCD, which we create CR in management cluster,

But for rolling out we need to create CR in member clusters, i am not sure how can we watch the member cluster CR's status

Reply:
Yes, this approach differs from FluxCD. 

It may not be feasible to directly monitor the status of Custom Resources (CRs) in member clusters. 

Instead, we might need to continuously iterate over these CRs, checking them until they achieve the expected state.

see: https://github.com/kurator-dev/kurator/blob/40eb58e05e3aa86c7df726dfebda54bc2070e76e/pkg/fleet-manager/backup/backup_controller.go#L166-L207